lib/web3/transfer.js

```diff
@@ -49,7 +50,7 @@ var transfer = function (from, iban, value, callback) {
         return deposit(from, address, value, icap.client());
     }
 
-    namereg.addr(icap.insitution(), function (err, address) {
+    namereg.addr(icap.institution(), function (err, address) {
         return deposit(from, address, value, icap.client(), callback);
```

Comment:
We should probably add a check for `if(err)` here, otherwise we send to `undefined`